.eslintignore

```diff
@@ -0,0 +1 @@
+reader-sdk-react-native-quickstart
```

Comment:
Did you mean to include this file?

Reply:
Mean exclude this and there is separate eslint config inside the project.